### PR TITLE
improve typehint coverage in utils.py

### PR DESCRIPTION
**Pull request by svenharris**
Aims to remove any strict MyPy errors from the utils.py file, relating to ticket #251

### CODE REVIEW

File: ogr/utils.py [hunk 1]
```diff
@@ -164,7 +166,7 @@ def __str__(self) -> str:
             f"RequestResponse("
             f"status_code={self.status_code}, "
             f"ok={self.ok}, "
-            f"content={self.content}, "
+            f"content={self.content.decode()}, "
```
Comment (lachmanfrantisek) [inline ogr/utils.py]: @jscotka is this ok? (I hope we don't break anything related to requre.)

Comment (jscotka) [reply ogr/utils.py]: should be okay requre should be trasparent.

Comment (lachmanfrantisek) [reply ogr/utils.py]: jscotka said it should be OK..

**Comment by lachmanfrantisek** [comment]
recheck

**Comment by lachmanfrantisek** [comment]
Please, rebase so we can merge that.

**Pull request merged**